### DISCUSSION TITLE
OpenEvse don't work with disabled secuity test

### DISCUSSION THREAD

**Discussion by DylanVautrin** [OpenEVSE/open_evse#34] [Q&A]
Hello, i would like to use my OpenEVSE and disabled GFCI and Ground check, but when i disable it in the GUI, the charging session never start,  the OpenEVSE display "Charging" but i don't have any power 
Can you help me please ? 
Best regards

**Comment by chris1howell** [comment]
If OpenEVSE displays Charging, it has done all it can do. Now it is up to
the vehicle to pull power.

**Reply by DylanVautrin** [reply]
Hello, thanks for replying.
I'm working with a M3 simulator, and usually, it works well when the test is enabled, and is only when i disabled it that it don't have any power, i'm in a particular case where i HAVE to disable the test, in particular the GFCI and Ground monitor.
Can you help me ?
